coalib/settings/ConfigurationGathering.py

```diff
@@ -147,7 +147,19 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
     for item in list(cli_sections["default"].contents.pop("targets", "")):
         targets.append(item.lower())
 
-    if bool(cli_sections["default"].get("no_config", "False")):
+    no_config_file_options = (
+        'no_config',
+        'show_bears',
+        'show_capabilities')
+
+    try:
+        do_not_load_config = any(
+            bool(cli_sections['default'].get(option, False))
+            for option in no_config_file_options)
+    except ValueError:
+        do_not_load_config = True
```

Comment:
Agree, it's look ugly, but it's cover cases whit 'coala -S no_config=True', and case with --show-capabilities argument (because it's not flag and can't be converted to bool)...